orm/db.go

```diff
@@ -621,6 +621,27 @@ func (d *dbBase) Update(q dbQuerier, mi *modelInfo, ind reflect.Value, tz *time.
 		return 0, err
 	}
 
+	var find bool
+	var index int
+	for i, col := range setNames {
+		if mi.fields.GetByColumn(col).autoNowAdd {
+			index = i
+			find = true
+		}
+	}
+
+	if find {
+		newSetNames := make([]string, 0, len(setNames)-1)
+		newSetNames = append(newSetNames, setNames[0:index]...)
+		newSetNames = append(newSetNames, setNames[index+1:]...)
+		setNames = newSetNames
```

Comment:
If you want to delete an element from a slice, you just can use `a = append(a[:i], a[i+1:]...)`

Reply:
thanks ,I have updated.